src/docfx/build/dependency/DependencyResolver.cs

```diff
@@ -245,5 +238,17 @@ public string GetRelativeUrl(Document fileRelativeTo, Document file)
 
             return (file != null ? null : Errors.FileNotFound(relativeTo.ToString(), path), file, null, query, fragment, false, pathToDocset);
         }
+
+        private string ResolveToRelativePath(string path, Document relativeTo)
+        {
+            foreach (var (alias, aliasPath) in relativeTo.Docset.ResolveAlias)
+            {
```

Comment:
assert that `alias` MUST ends with '/' otherwise `StartsWith` check is not sufficient.